src/pylorax/api/queue.py

```diff
@@ -249,15 +253,16 @@ def compose_detail(results_dir):
 
     * id - The uuid of the comoposition
     * queue_status - The final status of the composition (FINISHED or FAILED)
-    * timestamp - The time of the last status change
+    * job_created - When the user submitted the compose
+    * job_started - Anaconda started running
+    * job_finished - Job entered FINISHED or FAILED state
     * compose_type - The type of output generated (tar, iso, etc.)
     * blueprint - Blueprint name
     * version - Blueprint version
     * image_size - Size of the image, if finished. 0 otherwise.
     """
```

Comment:
Could you add a note here about the timestamps being Unix UTC timestamps, and that any or all of them could be None (or null in JSON) if they haven't been created yet.